redhat-developer/app-services-sdk-python
# fix: fixes some issues for the SDK generator

These are the edits that fix the Python SDK generator. 

In particular the edits to the `create_topic` section and refs which stem from there.

to verify:
Pull this OAS file and run `make generate`. The SDKs should be built without any issues
